libs/wingc/src/type_check/jsii_importer.rs

```diff
@@ -60,13 +59,13 @@ impl<'a> JsiiImporter<'a> {
 					_ => panic!("TODO: handle primitive type {}", primitive_name),
 				}
 			} else if let Some(Value::String(type_fqn)) = obj.get("fqn") {
-				if type_fqn == WINGSDK_INFLIGHT_FQN {
+				if type_fqn == &format!("{}.{}", WINGSDK_ASSEMBLY, WINGSDK_INFLIGHT) {
```

Comment:
Since the constants are static and `format!` is a macro, there's no runtime cost for this right?

Reply:
I'm almost certain there actually is runtime cost here because the formatting macro does need to create a string and concat all this together. I was considering [const_format](https://github.com/rodrimati1992/const_format_crates/) to avoid this, but then decided not... WDYT should I pull that crate in?

Reply:
probably an unneeded micro-optimization, I'm ok with it as-is